sdk/openai/openai/sources/customizations/api/getSSEs.browser.ts

```diff
@@ -14,7 +14,45 @@ export async function getSSEs(
 async function getStream<TResponse>(
   response: StreamableMethod<TResponse>
 ): Promise<ReadableStream<Uint8Array>> {
-  const stream = (await response.asBrowserStream()).body;
-  if (!stream) throw new Error("No stream found in response. Did you enable the stream option?");
-  return stream;
+  const { body, status } = await response.asBrowserStream();
+  if (status !== "200" && body !== undefined) {
+    const text = await streamToText(body);
+    throw JSON.parse(text).error;
```

Comment:
Would we want to wrap this message or whatever in an error object or custom error object?